### PR TITLE
add synonym for "shell" on windows: Command Prompt

### PR DESCRIPTION
**Pull request by daattali**

### CODE REVIEW

File: 80_shell.Rmd [hunk 1]
```diff
@@ -6,7 +6,7 @@ Even if you do most of your Git operations via a client, such as RStudio or GitK
 
 ## What is the shell?
 
-The `shell` (or `bash` or terminal) is a program on your computer whose job is to run other programs, rather than do calculations itself. The `shell` is a very old program and in a time before the mouse it was the only way to interact with a computer. It is still extremely popular among programmers because it is very fast, concise, and is particularly powerful at automating repetitive tasks.
+The `shell` (or `bash` or terminal or Command Prompt on Windows) is a program on your computer whose job is to run other programs, rather than do calculations itself. The `shell` is a very old program and in a time before the mouse it was the only way to interact with a computer. It is still extremely popular among programmers because it is very fast, concise, and is particularly powerful at automating repetitive tasks.
```
Comment (jennybc) [inline 80_shell.Rmd]: Good idea. Sure. Do you think we need to worry about bash vs cmd.exe here? Will the commands on this page work for a Windows user that hasn't installed git bash yet and truly is using cmd.exe? And then there's PowerShell too. Isn't that the default "shell" for newer versions of Windows? Maybe we should have a paragraph at the bottom reviewing the Windows situation?

Comment (daattali) [inline 80_shell.Rmd]: Some of the commands have different name in the windows shell. But I strongly think nobody should ever use the real windows shell, so maybe there can just be a note saying that they should only access the shell from "git bash" or from RStudio

Comment (jennybc) [inline 80_shell.Rmd]: So will you add some verbiage that makes it clear we're talking about bash? And if a windows user is getting puzzling results, that suggests they haven't installed git bash yet? And link to those install instructions?

@kevinushey tells me RStudio will launch cmd.exe if there's no other option (also seems to be related whether current project is a git repo?), so I don't think we can assume that ALL shells launched via RStudio are bash. Early on, while someone is still setting up, it is possible they will end up in cmd.exe.

Comment (daattali) [inline 80_shell.Rmd]: New PR coming up (I made this PR through the web interface, not sure if I can "ammend" to it?)

**Comment by jennybc** [comment]
So we close this one, right? Because the other supercedes it?

**Comment by daattali** [comment]
Yes

**Pull request closed without merging**